### ISSUE TITLE
shape-json ignore string ""

### ISSUE THREAD

**Issue by he-yang** [ansteh/shape-json#9]
In JSON, sometimes we would have something like this.

{"Name": "David", "Age":20, "Comment":""}

In the above JSON , the comment field is a string type and there is nothing in it.

Shape-json ignores "" by default, is there a way to have "" included.

**Comment by ansteh** [comment]
Hi Yang,
confirmed, i will fix it

**Comment by ansteh** [comment]
This test works now: [test code in repo](https://github.com/ansteh/shape-json/blob/master/test/spec/parse-string.js)
```js
    let input = [
      {"Name": "Tom", "Age":20, "Comment":""},
      {"Name": "Jerry", "Age":18, "Comment":"Catch me if you can!"},
    ]

    let scheme = {
      "$mirror[people](Name)": {
        "name": "Name",
        "age": "Age",
        "comment": "Comment"
      }
    };

    let result = {
      people: [
        {name: "Tom", age: 20, comment: ''},
        {name: "Jerry", age: 18, comment: 'Catch me if you can!'},
      ]
    };

    // console.log(JSON.stringify(shape.parse(input, scheme), null, 2));

    expect(shape.parse(input, scheme)).toEqual(result);// true

```

**Comment by he-yang** [comment]
Thank you Andre for the prompt response.
 
 
------------------ Original ------------------
From: "Andre Stehle"; 
Date: 2017年7月13日(星期四) 凌晨2:39
To: "ansteh/shape-json"; 
Cc: "He Yang"; "Author"; 
Subject: Re: [ansteh/shape-json] shape-json ignore string "" (#9)

 

Closed #9.
